we have the system library and ctypes.util.find_library("tdjson") finds it
correctly, the fallback to pkg_resources isn't used (and wouldn't work anyway
as that only provides linux/darwin libraries so they're removed)

Index: telegram/tdjson.py
--- telegram/tdjson.py.orig
+++ telegram/tdjson.py
@@ -5,8 +5,6 @@ import ctypes.util
 from ctypes import CDLL, CFUNCTYPE, c_int, c_char_p, c_double, c_void_p, c_longlong
 from typing import Any, Dict, Optional, Union
 
-import pkg_resources
-
 logger = logging.getLogger(__name__)
 
 
